core/embed/rust/trezorhal.h

```diff
@@ -0,0 +1,2 @@
+#include "secbool.h"
+#include "usb.h"
```

Comment:
Missing newline at the end of file. (Maybe your editor does not recognize `.editorconfig ` ?)